### PR TITLE
V2Feature - TRSV2-203 : Login Journey

### PR DESCRIPTION
**Pull request by chris-pettinga**

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -111,7 +111,7 @@ six==1.16.0
     #   python-dateutil
 sqlparse==0.4.2
     # via django
-trade-remedies-client @ git+https://github.com/uktrade/trade-remedies-client.git
+trade-remedies-client @ git+https://github.com/uktrade/trade-remedies-client.git@v2feature/trsv2-174/login-journey
```
Comment (KnowYourLines) [inline requirements/base.txt:114]: This probably needs updating

File: requirements/dev.txt [hunk 2]
```diff
@@ -290,7 +290,7 @@ tomli==2.0.1
     #   coverage
     #   pep517
     #   pyproject-flake8
-trade-remedies-client @ git+https://github.com/uktrade/trade-remedies-client.git
+trade-remedies-client @ git+https://github.com/uktrade/trade-remedies-client.git@v2feature/trsv2-174/login-journey
```
Comment (KnowYourLines) [inline requirements/dev.txt:293]: Same as above

File: requirements/prod.txt [hunk 3]
```diff
@@ -138,7 +138,7 @@ sqlparse==0.4.2
     # via
     #   -r requirements.in/../requirements/base.txt
     #   django
-trade-remedies-client @ git+https://github.com/uktrade/trade-remedies-client.git
+trade-remedies-client @ git+https://github.com/uktrade/trade-remedies-client.git@v2feature/trsv2-174/login-journey
```
Comment (KnowYourLines) [inline requirements/prod.txt:141]: Likewise

File: requirements.in/base.in [hunk 4]
```diff
@@ -1,4 +1,4 @@
-git+https://github.com/uktrade/trade-remedies-client.git
+git+https://github.com/uktrade/trade-remedies-client.git@v2feature/trsv2-174/login-journey#trade-remedies-client
```
Comment (KnowYourLines) [inline requirements.in/base.in:1]: Should update this before merging

**Review by KnowYourLines — CHANGES_REQUESTED** [review]

**Comment by KnowYourLines** [comment]
LGTM client library requirement just needs updating

File: requirements/base.txt (hunk 1 above)
Comment (chris-pettinga) [reply requirements/base.txt:114]: Same comment for the below

**Review by KnowYourLines — APPROVED** [review]

**Comment by KnowYourLines** [comment]
Leaving this pointing to the https://github.com/uktrade/trade-remedies-client/pull/12 unmerged until this is tested in UAT and ready for prod release as merging https://github.com/uktrade/trade-remedies-client/pull/12 would break current production journeys

**Pull request merged**